db/schema.rb

```diff
@@ -10,16 +10,35 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_08_172657) do
+ActiveRecord::Schema[7.0].define(version: 2023_09_11_163456) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "customer_items", force: :cascade do |t|
+    t.bigint "customer_id", null: false
+    t.bigint "item_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["customer_id"], name: "index_customer_items_on_customer_id"
+    t.index ["item_id"], name: "index_customer_items_on_item_id"
+  end
+
+  create_table "customers", force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "item_id", null: false
```

Comment:
To get rid of this, you can do a migration to remove the column. 

Since it's an IC, the faster option might be do to do 
1)`rails db:{drop,create}` to reset the db
2) delete the migrations that add references to the customer and item table individually (the ones that I left a comment on, keep the customer migration and the one creating the customer_items joins table)
3) run `rails db:migrate`

This is not best practice if this were an application you're working on with someone else, or if you were at a job, but for an IC, this is a fast way to deal with this issue!